### ISSUE TITLE
docs are out of date with corresponding code

### ISSUE THREAD

**Issue by BIRooter** [kernelci/kernelci-core#2040]
Docs are out of date with corresponding code.  

kci_test.md has wrong parameters that aren't supported by kci_test.

Could you update them, so that users needn't to analyze code to find proper parameters?

**Comment by gctucker** [comment]
Yes, it's part of the command line tools that are going to be deprecated but we should keep supporting it for at least 6 months so it makes sense to fix that.  I believe the `--runtime-config` parameter is the main change, was there anything else that caused you some trouble?

**Comment by gctucker** [comment]
Some sample commands to use `kci_test` can be found here by the way:
https://github.com/kernelci/kernelci-core/pull/2023

That's not real documentation but it can be used as a reference when updating the documentation.

**Comment by nuclearcat** [comment]
Reviewed against current main. This issue concerns kci_test, whose executable and implementation were removed with the legacy workflow in https://github.com/kernelci/kernelci-core/commit/4e6f088a. The old documentation was moved under doc/deprecated by https://github.com/kernelci/kernelci-core/commit/2ffaf6de. There is no longer a supported kci_test interface for these pages to document, so closing as obsolete.
